Automattic/jetpack
# Connection: Add rate limiter to the package versions endpoint calls

## Proposed changes:
* Limit the package versions API calls to one per minute.

### Other information:


## Jetpack product discussion
p1706561650832669-slack-C05PV073SG3

## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:
1. Connect Jetpack, go to Jetpack Dashboard and confirm the package versions were sent in Kibana's nginx logs (replace the blog ID with yours in c06aef671133a135b190d14bcb71a6f3-logstash).
2. Delete the option `wp option delete jetpack_package_versions` and refresh Jetpack Dashboard. Repeat a bunch of times withing the next couple of minutes. After each attempt run `wp option get jetpack_package_versions`: empty option should mean a request has already been sent in the last minute. If the option got filled up, the request has been sent (that should happen not sooner than a minute after the last request).
4. Check Kibana again and confirm only one request per minute is being sent.
